src/ignitus-Dashboard/ignitus-Dashboard-Header/Styles/style.scss

```diff
@@ -0,0 +1,230 @@
+@import './src/ignitus-Shared/styles/ignitus-tokens';
+
+
+
+.nav-item a{
+font-family: Raleway;
+font-style: normal;
+font-weight: bold;
+text-align: center;
+color: $ignite-color-blue-bg;
+}
+
+.home {
+  position: absolute;
```

Comment:
Please avoid using `position absolute` everything can be done using `flexbox`, until and unless you plan to use `z-index`.